QgisModelBaker/gui/validate.py

```diff
@@ -464,3 +475,10 @@ def _get_feature_in_project(self, t_ili_tid):
                     if feature.attributes()[idx] == t_ili_tid:
                         return layer, feature
         return None, None
+
+    def _select_validator_config_file(self):
+        filename, _ = QFileDialog.getOpenFileName(
```

Comment:
You can use here make_file_selector from qt_utils.py doing not much but having less lines of code here.

Anyway, would be nice to pass a filter on "ini" and "INI".

Reply:
Thanks I changed that as suggested